specification/sql/resource-manager/readme.md

```diff
@@ -24,13 +24,115 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-composite-v3
+tag: package-composite-v4
 ```
 
 ## Composite packages
 
 The following packages may be composed from multiple api-versions.
 
+### Tag: package-composite-v4
+
+These settings apply only when `--tag=package-composite-v4` is specified on the command line.
+
+This section contains the "composite-v4" set of APIs, which is composed from a selection of api-versions that will remain backwards compatible with "v4" clients such as .NET SDK Microsoft.Azure.Management.Sql version 1.44.3.0-preview.
+
+APIs must only be added to this section when the API is publicly available in at least 1 production region and at least 1 generated client has been tested end-to-end.
+
+Differences in v4 (compared to v3):
+
+* Added new API for databases and elastic pools
+
+  * `-2020-08-01/databases.json`
+
+  * `-2020-08-01/elasticPools.json`
```

Comment:
Could you please add that failover API for elastic pool was integrated into elasticPools? (if you think this is reasonable)